### PR TITLE
Add Nimbox

### PR DESCRIPTION
**Pull request by ghost**
Nimbox meets criteria 2 (data).

### CODE REVIEW

File: _data/backup.yml [hunk 1]
```diff
@@ -116,6 +116,14 @@ websites:
     twitter: MEGAprivacy
     img: mega.png
     tfa: No
+
```
Comment (Jawshy) [inline _data/backup.yml]: Please remove these spaces.

**Review by Jawshy — CHANGES_REQUESTED** [review]
Hi, @tomchappelow. Thanks for submitting your site for listing on twofactorauth.org. 

I noticed that the support page indicates email 2FA as an option. Could you add the following under line 124? 

```ruby
    email: Yes
```

**Comment by ghost** [comment]
Thanks for reviewing this, @Jawshy. Change added.

**Comment by Jawshy** [comment]
Thanks. 

_(Referencing https://github.com/2factorauth/twofactorauth/pull/2285)_

**Review by Jawshy — APPROVED** [review]

**Comment by Carlgo11** [comment]
@tomchappelow why did you create both #2286 and #2285?

**Review by Carlgo11 — APPROVED** [review]

**Comment by ghost** [comment]
@Carlgo11 desktop git issues. But, I closed #2285 just after creating it.

**Pull request merged**